crates/bench/benches/uv.rs

```diff
@@ -148,13 +137,21 @@ mod resolver {
         let index = InMemoryIndex::default();
         let index_locations = IndexLocations::default();
         let installed_packages = EmptyInstalledPackages;
-        let interpreter = venv.interpreter();
-        let python_requirement = PythonRequirement::from_interpreter(interpreter);
         let sources = SourceStrategy::default();
-
         let options = OptionsBuilder::new().exclude_newer(exclude_newer).build();
         let build_constraints = [];
 
+        let tags = interpreter.tags().unwrap();
+        let markers = interpreter.markers();
```

Comment:
I think this means the benchmark will vary by interpreter rather than installing for a specific, consistent platform. Is that intentional?

Reply:
Right right, reverted that change.